libr/core/xrefs.c

```diff
@@ -0,0 +1,53 @@
+/* radare - LGPL - Copyright 2009-2014 - pancake, nibble, Darredevil */
+
+#include <r_core.h>
+#include <sdb.h>
+
+#define _DB core->anal->sdb_xrefs
+
+R_API bool r_core_xrefs_load(RCore *core, const char *prjfile) {
+    char *path, *db;
+    ut8 found = 0;
+    SdbListIter *it;
+    SdbNs *ns;
+
+    const char *prjdir = r_config_get (core->config, "dir.projects");
+
+    if (!prjfile || !*prjfile) {
+        return false;
+    }
+
+    if (prjfile[0] == '/') {
+        db = r_str_newf ("%s.d", prjfile);
+        if (!db) return false;
+        path = strdup (db);
+    } else {
+        db = r_str_newf ("%s/%s.d", prjdir, prjfile);
+        if (!db) return false;
+        path = r_file_abspath (db);
+    }
+
+    if (!path) {
+        free (db);
+        return false;
+    }
+
+    ls_foreach (core->anal->sdb->ns, it, ns){
```

Comment:
this should be implemented in a new API in sdb.ns, mind to send a PR in sdb to extend that functionality? sdb_ns_unset seems the right name for it and just return bool.. well while explaininng it i have already implemented and updated it